src/extension.ts

```diff
@@ -88,6 +135,171 @@ export class ${className} extends ${className}Dto {
 	return fileContents;
 }
 
+function generateBackendService(className: string, namespaceName: string): string {
+	/* TODO
+	 * 1. Fix assumptions about namespace structure (described in interface function)
+	 * 2. Find the actual DB name in the DBContext
+	 * 		a. Even if there are multiple DBContexts we should be able to use the passed in DBContext class name. 
+	 * 			Maybe instead of having a DBContext name and namespace we should have a file path to it.
+	 * 		b. Need to determine what to do if the are multiple DBSets for 1 class
+	 * 3. Determine the Primary Key from the C# model
+	 * 4. Generate Request classes based on the C# model
+	 * 		a. Might need to add in a configruation on if we want to include children in the Request model
+	 * 5. Add exceptions for Not Found responses
+	 * 6. Will probably need to fix the assumption that we can just use Microsoft.EntityFrameworkCore as the DB context
+	 * 7. Need to consider what to do if multiple classes in same file when scaffolding CRUD, currently just take the first
+	 * 8. Need to consider where AutoMapper file lives
+	 * 9. Need to consider where to add service injection
```

Comment:
More considerations:
1. Filename suffix schemes (Controller vs Service vs RequestHandler, etc)
2. Logging
3. Whether or not there are separate models for repo and db objects
4. Binding models vs dtos

Reply:
Added these